_includes/download_details_pc.md

```diff
@@ -0,0 +1,7 @@
+Installable live images support a local installation (with the included packages) or a network installation (packages are downloaded from official repository).
+
+Log in as `anon`/`root`, password `voidlinux`.
```

Comment:
In other places in our documentation we refer to user/pass combos with the slash, and this could be confusing.  Please revise to `anon/voidlinux`, `root/voidlinux`.